### PR TITLE
layer.conf: Define LAYERDEPENDS

### PR DESCRIPTION
**Pull request by agherzan**

### CODE REVIEW

File: conf/layer.conf [hunk 1]
```diff
@@ -10,6 +10,7 @@ BBFILE_PATTERN_raspberrypi := "^${LAYERDIR}/"
 BBFILE_PRIORITY_raspberrypi = "9"
 
 LAYERSERIES_COMPAT_raspberrypi = "hardknott honister"
+LAYERDEPENDS_raspberrypi = "core"
```
Comment (kraj) [inline conf/layer.conf:13]: dont me require meta-python and meta-oe as well ?

Comment (agherzan) [reply conf/layer.conf:13]: I'll give it a sping but I think we isolated those deps. Let's see...

Comment (agherzan) [reply conf/layer.conf:13]: Parsing succeeds on `core-image-base` without anything but:

```
layer                 path                                      priority
==========================================================================
meta                  /home/andrei/work/yocto/poky/meta         5
meta-poky             /home/andrei/work/yocto/poky/meta-poky    5
meta-yocto-bsp        /home/andrei/work/yocto/poky/meta-yocto-bsp  5
meta-raspberrypi      /home/andrei/work/yocto/meta-raspberrypi  9
```

Comment (kraj) [reply conf/layer.conf:13]: does it build too ?

Comment (agherzan) [reply conf/layer.conf:13]: 2003/4240

What what do you suspect would be the problem? I would have expected parsing to surface a dangling bbappend or so.

Comment (agherzan) [reply conf/layer.conf:13]: All good on master.

**Review by kraj — APPROVED** [review]
LGTM

**Pull request merged**